spec/facility_spec.rb

```diff
@@ -23,4 +24,252 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe '#add facility' do
+    it 'can add facilities' do
+      facility_1 = Facility.new({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
+      facility_2 = Facility.new({name: 'DMV Northeast Branch', address: '4685 Peoria Street Suite 101 Denver CO 80239', phone: '(720) 865-4600'})
+      expect(facility_1).to be_a Facility
+      expect(facility_2).to be_a Facility
+    end
+  end
```

Comment:
I'm a little confused as to what we are testing in this block. The description says that you're testing the ability to add facilities but I don't see that reflected in the block. These tests look exactly the same as the assertion on line `10`.